pkg/kubeapiserver/options/authentication.go

```diff
@@ -397,16 +400,51 @@ func (o *BuiltInAuthenticationOptions) ToAuthenticationConfig() (kubeauthenticat
 		}
 	}
 
-	if o.OIDC != nil {
-		ret.OIDCCAFile = o.OIDC.CAFile
```

Comment:
I am wondering if we can update `OIDCAuthenticationOptions` to contain a `JWTAuthenticator` and then validate is here:

https://github.com/kubernetes/kubernetes/blob/1db4658614023c3630a2e8d9a634eec4980d074f/pkg/kubeapiserver/options/authentication.go#L186-L191